terminal.go

```diff
@@ -464,10 +468,18 @@ func (g *LightningTerminal) start() error {
 	case <-readyChan:
```

Comment:
I think we need to react to the `unlockChan` here somehow. I tested integrated mode where `lnd` was locked, and in that state it shows `running: false` for `lnd`. And I can't run `lncli --rpcserver localhost:8443 state` because I get `rpc error: code = Unknown desc = lnd is not running:`. Can we maybe count `lnd` as being locked as a successful startup and let `RPC` calls through? Otherwise we also won't be able to create a wallet or unlock using the 8443 port.

Reply:
Also, again same issue with not being able to shut down while lnd is not unlocked yet. Maybe something small after all?

Reply:
good catch! I have moved  the `g.statusMgr.SetRunning(subservers.LND)` line a bit earlier 👍

Reply:
this also pointed to me to another pre-existing issue which is that LND's whitelisted calls are currently still checked for macaroons if they go through 8443. So i'll open another PR to fix that too :)

Reply:
ok - added this PR: https://github.com/lightninglabs/lightning-terminal/pull/617

state requests should now work even while LND is still locked 👍